test/raft/replication.hh

```diff
@@ -1235,7 +1235,7 @@ future<> raft_cluster<Clock>::check_rpc_config(::check_rpc_config cc) {
     for (auto& node: cc.nodes) {
         BOOST_CHECK(node.id < _servers.size());
         co_await seastar::async([&] {
-            CHECK_EVENTUALLY_EQUAL(_servers[node.id].rpc->known_peers(), as);
+            BOOST_CHECK(eventually_true([&] { return _servers[node.id].rpc->known_peers() == as; }));
```

Comment:
Why not use the new function here too?

Reply:
Because BOOST_REQUIRE_EQUAL doesn't know how to print `raft::server_address_set`